packages/tools/pluggable-widgets-tools/configs/rollup-plugin-collect-dependencies.js

```diff
@@ -16,28 +23,58 @@ export function collectDependencies({ onlyNative, outputDir, widgetName }) {
             managedDependencies.length = 0;
         },
         async resolveId(source, importer) {
-            if (source.startsWith(".") || source.startsWith("/")) {
+            // eslint-disable-next-line no-control-regex
+            const sourceCleanedNullChar = source.replace(/\x00/g, "");
```

Comment:
The `source` must be as written in the import statement. (You can check [here](https://rollupjs.org/guide/en/#resolveid) for more information about the resolveId hook) I don't know the reason but sometimes the `source` starts with a `null` character (\x00). So the following if statement doesn't work as expected because of the null char.